controllers/useraccount/useraccount_controller_test.go

```diff
@@ -1050,6 +1051,89 @@ func TestReconcile(t *testing.T) {
 		err = r.Client.Get(context.TODO(), types.NamespacedName{Name: userAcc.Name}, user)
 		require.NoError(t, err)
 	})
+
+	// Test UserAccount with OriginalSub property set
+	// TODO remove this test after migration is complete
+	t.Run("create or update identities from OriginalSub OK", func(t *testing.T) {
+		userAcc := newUserAccount(username, userID, false)
+		userAcc.Spec.OriginalSub = fmt.Sprintf("original-sub:%s", userID)
+
+		t.Run("create user identity mapping", func(t *testing.T) {
+			r, req, _, _ := prepareReconcile(t, username, userAcc, preexistingUser)
+			//when
+			res, err := r.Reconcile(context.TODO(), req)
+
+			//then
+			require.NoError(t, err)
+			assert.Equal(t, reconcile.Result{}, res)
+
+			updatedUser := &userv1.User{}
+			err = r.Client.Get(context.TODO(), types.NamespacedName{Namespace: "", Name: userAcc.Name}, updatedUser)
+			require.NoError(t, err)
+
+			t.Run("create first identity", func(t *testing.T) {
+				r, req, _, _ := prepareReconcile(t, username, userAcc, updatedUser)
```

Comment:
you should use the same reconcile instance from the previous test. The same applies also to the FakeClient (that is created as part of this function).
So please remove the line 1075
```
r, req, _, _ := prepareReconcile(t, username, userAcc, updatedUser)
```

Reply:
You're right, these aren't necessary - however this entire subtest is going away once migration is complete, and these extra lines don't break the test.  Do you think it's important we devote the time to clean up this minor irregularity instead of working on more important issues?

Reply:
Yeah.. I noticed a few things in that code that I would do differently if it was a permanent code but since it's all about a very temporal migration code I think it's OK if it's not super clean if we are sure if it functions as expected.

Reply:
> however this entire subtest is going away once migration is complete

then please add a comment saying that to this part of the code next time. You did that in the previous PRs, and that was really helpful.

>  and these extra lines don't break the test

they don't break the test, but the reliability of the test is lower

>  Do you think it's important we devote the time to clean up this minor irregularity instead of working on more important issues?

If you waited one more day for my review, then addressing my comments would be as simple as clicking on the green button that would accept my suggestion. Not time extra and you could focus on the more important issues.
The PR was there for a month. I waited for your responses for two weeks. You asked me for a review and I answered that I would review it. This PR was not urgent at all, so waiting one more day wouldn't kill us.